Retarus/retarus_modules/retarus_events_comsumer.py

```diff
@@ -0,0 +1,63 @@
+from queue import SimpleQueue
+from threading import Event, Thread
+import time
+
+import orjson
+from sekoia_automation.connector import Connector
+from websocket import WebSocketApp, WebSocketTimeoutException
+
+
+class RetarusEventsConsumer(Thread):
+    def __init__(self, connector: Connector, queue: SimpleQueue):
+        super().__init__()
+        self.connector = connector
+        self.queue = queue
+        self.configuration = connector.configuration
+        self._stop_event = Event()
+
```

Comment:
Called by `RetarusConnector`

```suggestion

    def stop(self):
        self._stop_event.set()
```